java/test.sh

```diff
@@ -75,8 +75,15 @@ popd
 
 
 pushd "$ROOT_DIR"
+
+echo "Check java code format."
+# check google java style
+mvn -T16 spotless:check
+# check naming and others
+mvn -T16 checkstyle:check
```

Comment:
Can we move these to the very beginning of this file?

Reply:
Will do

Reply:
fixed